tests/system/providers/amazon/aws/example_bedrock_knowledge_base.py

```diff
@@ -0,0 +1,519 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Some of this needs adjusting, templated fields aren't working right with the way I wrote this.  Fix is coming,

Reply:
done